common/api-review/firestore-lite.api.md

```diff
@@ -230,7 +230,7 @@ export abstract class QueryConstraint {
 }
 
 // @public
-export type QueryConstraintType = 'where' | 'orderBy' | 'limit' | 'limitToLast' | 'startAt' | 'startAfter' | 'endAt' | 'endBefore';
+export type QueryConstraintType = 'where' | 'orderBy' | 'limit' | 'limitToLast' | 'startAt' | 'startAfter' | 'endAt' | 'endBefore' | 'and' | 'or';
```

Comment:
given that we are not exposing 'and' and 'or' publicly, we should remove this update to the .md file

Reply:
oops.. I thought if something is not exported in [api.ts](https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/src/api.ts) then it's not considered public API.

Reply:
Ah! Got it. Thanks

Reply:
Thanks for the feedback, but I don't think there's any value in trying to address this. Although this addition of 'and' and 'or' in `QueryConstraintType` will be public, it does not make this feature usable. In fact, I'm not sure why `QueryConstraintType` is publicly exposed at all, but given that it is, I'm not going to modify that.

So, @hsubox76, @ehsannas, I'm suggesting to move forward with this PR as is.

Reply:
Merging to a feature branch so we can avoid any public API changes until the feature is released.